### ISSUE TITLE
RemoveTimeBucket access to delete errant data loads

### ISSUE THREAD

**Issue by bdowling** [alpacahq/marketstore#102]
There does not appear to be a way to access RemoveTImeBucket in the RPC API, which is probably good from a security standpoint.  Perhaps there should be a CLI way to access it?

Based on the simple file system layout I assumed I could remove the files and restart the service.

**Comment by umitanuki** [comment]
Good find, I thought we had an issue filed here.  The original closed-source version had it and we simply didn't have time to add RPC for the remove feature. If you talk about security, the write interface is already exposed, so I don't mind exposing the remove via RPC as well.

**Comment by bdowling** [comment]
fwiw, I had just also noticed the `enable_add` and `enable_remove` configuration flags, however I do not see where these are ever used in any frontend code..

**Comment by umitanuki** [comment]
Yes those flags used to be used to protect accidental data write but not now.  Probably should be removed.  The current assumption is that you know what you are doing, and the server should not be exposed to public.  if you need authentication, have nginx or something in between.  That said, as the product matures, designing correct auth feature isn't a bad idea.

**Comment by umitanuki** [comment]
A bit of more history is that the write RPC was added first, and we started to populate data using the interface, and it was very dangerous as dev/staging/production mixed by accident.  This was before the data ingestion bgworker was introduced. Nowadays, only those bgworkers are writing the data, most if not all the cases, so it may make sense to restrict write/remove RPC as well and open it only when turned on in configuration, or cli.
